README.md

```diff
@@ -123,7 +123,7 @@ Another take on this idea from Stu Robson: https://github.com/sturobson/myRSS
 - [steveedson.co.uk](https://steveedson.co.uk/) [[RSS](https://steveedson.co.uk/feed.xml)]
 - [Simurai](http://simurai.com/)
 - [Sushi and Robots by Jina Anne](https://www.sushiandrobots.com/)
-- [Suze Shardlow](https://suze.dev/)
+- [Suze Shardlow](https://suze.dev/) [RSS](https://suze.dev/feed.xml)
```

Comment:
Hi Suze! Awesome work 🎉

Would you amending slightly so it follows the format of the others?

```suggestion
- [Suze Shardlow](https://suze.dev/) [[RSS](https://suze.dev/feed.xml)]
```